package-testing/pre/000_install_package.rb

```diff
@@ -1,18 +1,42 @@
+require 'open-uri'
+require 'json'
+
 test_name 'Install pdk package on workstation host' do
   workstation = find_at_most_one('workstation')
 
   step 'Install pdk package' do
     if ENV['LOCAL_PKG']
       pkg = File.basename(ENV['LOCAL_PKG'])
       scp_to(workstation, ENV['LOCAL_PKG'], pkg)
-    end
+    else
```

Comment:
This `else` looks wrong - if you're using `LOCAL_PKG` then you still need the build version string for generic_install_dmg to get the pkg name inside the dmg